### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by UlybinSergej**

### CODE REVIEW

File: src/task3/task1/Person.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package task3.task1;
+
+
+public class Person {
+    private String firstName;
+    private String lastName;
+    private String sex;
+
+    public Person() {
+    }
```
Comment (YanBelikov) [inline src/task3/task1/Person.java]: Why are you use it here?

File: src/task3/task1/PersonMain.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package task3.task1;
+
+import java.util.Random;
+
+public class PersonMain {
+    public static void main(String[] args) {
+        String[][] firstNameArray = {{"Михаил", "Мария", "Андрей", "Юлия", "Кристина"}, {"m", "f", "m", "f", "f"}};
+        String[] lastNameArray = {"Петров", "Сидоров", "Иванов", "Смирнов", "Кузнецов", "Попов"};
+        Person[] persons = new Person[50];
```
Comment (YanBelikov) [inline src/task3/task1/PersonMain.java]: it's magic number
Please use variables for anything numbers exclude 0 and 1

File: src/task3/task1/PersonMain.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+        Random random = new Random();
+
+        for (int i = 0; i < 50; i++) {
```
Comment (YanBelikov) [inline src/task3/task1/PersonMain.java]: use new variable for 50

File: src/task3/task1/PersonMain.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            int temp = random.nextInt(firstNameArray.length);
+            persons[i] = new Person(firstNameArray[0][temp], lastNameArray[random.nextInt(lastNameArray.length)], firstNameArray[1][temp]);
+            System.out.println("Человек №" + (i + 1) + " " + persons[i].toString());
```
Comment (YanBelikov) [inline src/task3/task1/PersonMain.java:17]: Why are you use here (i + 1)?

File: src/task3/task2/RandomMain.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        new SequentialRandom();
+        System.out.println(SequentialRandom.getRandomValue());
+        new SequentialRandom();
+        System.out.println(SequentialRandom.getRandomValue());
+        SequentialRandom.resetRandom();
+        new SequentialRandom();
+        System.out.println(SequentialRandom.getRandomValue());
+        new SequentialRandom();
+        System.out.println(SequentialRandom.getRandomValue());
+
```
Comment (YanBelikov) [inline src/task3/task2/RandomMain.java]: delete empty line

File: src/task3/task1/Person.java (hunk 1 above)
Comment (UlybinSergej) [reply src/task3/task1/Person.java]: Done

File: src/task3/task1/PersonMain.java (hunk 2 above)
Comment (UlybinSergej) [reply src/task3/task1/PersonMain.java]: Done

File: src/task3/task1/PersonMain.java (hunk 3 above)
Comment (UlybinSergej) [reply src/task3/task1/PersonMain.java]: Done

File: src/task3/task2/RandomMain.java (hunk 5 above)
Comment (UlybinSergej) [reply src/task3/task2/RandomMain.java]: Done

File: src/task3/task1/PersonMain.java (hunk 4 above)
Comment (UlybinSergej) [reply src/task3/task1/PersonMain.java:17]: I use this to correctly display objects from an array, because the  first object in the array has index zero

**Review by YanBelikov — APPROVED** [review]

**Pull request merged**